portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+function validateForm() {
+  const sNumber = document.getElementById("number").value;
+  var errorSpan = document.getElementById("error-message");
+  const regex = /^\d+$/;
+  errorSpan.innerText = "";
+  if (sNumber.length > 0) {
```

Comment:
It looks like this function will return undefined when `sNumber` is the empty string. This works as written since undefined is "falsy". To make it explicit, can you have `validateForm` return `false` when `sNumber` is empty?

Reply:
Fixed!